src/braket/circuits/circuit.py

```diff
@@ -438,6 +440,136 @@ def add_circuit(
 
         return self
 
+    def add_noise(
+        self,
+        noise: Noise,
+        target_gates: Union[str, Iterable[str]] = None,
```

Comment:
All of the parameters should be optional (e.g. target_gates: Optional[Union[str, Iterable[str]]]